### PR TITLE
Core/SAI: remove wrong DB errors.

### PR DESCRIPTION
**Pull request by grimgravy**
##### CHANGES PROPOSED:

-  remover os aviso constantes que aparece ao longo do server
-  

###### ISSUES ADDRESSED:
SmartScript: Entry 31105 has events but no events added to list because of instance flags.

Closes 

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

File: data/sql/updates/db_world/2018_10_17_01.sql [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+--
+-- END UPDATING QUERIES
+--
+COMMIT;
+END //
+DELIMITER ;
+CALL updateDb();
+DROP PROCEDURE IF EXISTS `updateDb`;
```
Comment (FrancescoBorzi) [inline data/sql/updates/db_world/2018_10_17_01.sql]: please read: https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files#how-to-create-an-sql-update-file

File: data/sql/updates/db_world/2018_11_04_00.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+(34453,1,0,68361),
+(34454,1,0,68361);
+--
+-- END UPDATING QUERIES
+--
+COMMIT;
+END //
+DELIMITER ;
+CALL updateDb();
+DROP PROCEDURE IF EXISTS `updateDb`;
```
Comment (FrancescoBorzi) [inline data/sql/updates/db_world/2018_11_04_00.sql]: same as above

File: src/server/game/Achievements/AchievementMgr.cpp [hunk 3]
```diff
@@ -2157,9 +2157,11 @@ void AchievementMgr::CompletedAchievement(AchievementEntry const* achievement)
         return;
 
 #if defined(ENABLE_EXTRAS) && defined(ENABLE_EXTRA_LOGS)
-    sLog->outDetail("AchievementMgr::CompletedAchievement(%u)", achievement->ID);
+        sLog->outDetail("AchievementMgr::CompletedAchievement(%u)", achievement->ID);
```
Comment (FrancescoBorzi) [inline src/server/game/Achievements/AchievementMgr.cpp]: bad indentation

File: src/server/game/Battlegrounds/Battleground.cpp [hunk 4]
```diff
@@ -815,6 +815,49 @@ void Battleground::EndBattleground(TeamId winnerTeamId)
...
+					sql_query << "INSERT INTO armory_game_chart (`gameid`,`teamid`,`guid`,`changeType`,`ratingChange`,`teamRating`,`damageDone`,`deaths`,`healingDone`,`damageTaken`,`healingTaken`,`killingBlows`,`mapId`,`start`,`end`) VALUES ('" << gameID << "', '" << resultTeamID << "', '" << player->GetGUID() << "', '" << changeType << "', '" << ratingChange << "', '" << resultRating << "', '" << itr->second->DamageDone << "', '" << itr->second->Deaths << "', '" << itr->second->HealingDone << "', '" << itr->second->DamageTaken << "', '" << itr->second->HealingTaken << "', '" << itr->second->KillingBlows << "', '" << m_MapId << "', '" << m_StartTime << "', '" << m_EndTime << "')";
+					CharacterDatabase.Execute(sql_query.str().c_str());
+				}
+				/** World of Warcraft Armory **/
+
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Battleground.cpp]: please fix your indentation

File: src/server/game/Battlegrounds/Battleground.cpp [hunk 5]
```diff
@@ -1399,6 +1442,14 @@ void Battleground::UpdatePlayerScore(Player* player, uint32 type, uint32 value,
...
             }
             break;
+			/** World of Warcraft Armory **/
+		case SCORE_DAMAGE_TAKEN:
+			itr->second->DamageTaken += value;              // Damage Taken
+			break;
+		case SCORE_HEALING_TAKEN:
+			itr->second->HealingTaken += value;             // Healing Taken
+			break;
+		/** World of Warcraft Armory **/
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Battleground.cpp]: please fix your indentation

File: src/server/game/Battlegrounds/Battleground.h [hunk 6]
```diff
@@ -216,6 +216,10 @@ enum ScoreType
     //SOTA
     SCORE_DESTROYED_DEMOLISHER  = 18,
     SCORE_DESTROYED_WALL        = 19,
+	
+	/** World of Warcraft Armory **/
+	SCORE_DAMAGE_TAKEN          = 20,
+	SCORE_HEALING_TAKEN         = 21
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Battleground.h]: please fix your indentation

File: src/server/game/Battlegrounds/Battleground.h [hunk 7]
```diff
@@ -252,7 +256,7 @@ enum BattlegroundStartingEventsIds
 struct BattlegroundScore
 {
     BattlegroundScore(Player* player) : KillingBlows(0), Deaths(0), HonorableKills(0), BonusHonor(0),
-        DamageDone(0), HealingDone(0), player(player)
+        DamageDone(0), HealingDone(0), DamageTaken(0), HealingTaken(0), player(player) //ARMORY MOD
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Battleground.h]: Armory Mod ?

File: src/server/game/Battlegrounds/Battleground.h [hunk 8]
```diff
@@ -263,6 +267,8 @@ struct BattlegroundScore
     uint32 BonusHonor;
     uint32 DamageDone;
     uint32 HealingDone;
+	uint32 DamageTaken;
+	uint32 HealingTaken;
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Battleground.h]: please fix your indentation

File: src/server/game/Battlegrounds/Zones/BattlegroundWS.h [hunk 9]
```diff
@@ -63,6 +63,16 @@ enum BG_WS_WorldStates
     BG_WS_STATE_TIMER_ACTIVE      = 4247
 };
 
+enum BG_WS_WinOrMapFlag
+{
+	BG_WS_NORMAL_FLAG_CAPTURE_REPUTATION          = 35,
```
Comment (FrancescoBorzi) [inline src/server/game/Battlegrounds/Zones/BattlegroundWS.h]: this line is not indented like the other ones

File: src/server/game/Entities/Item/Item.cpp [hunk 10]
```diff
@@ -266,6 +266,14 @@ bool Item::Create(uint32 guidlow, uint32 itemid, Player const* owner)
...
     SetUInt32Value(ITEM_FIELD_DURATION, itemProto->Duration);
     SetUInt32Value(ITEM_FIELD_CREATE_PLAYED_TIME, 0);
+	
+	if (itemProto->Quality > 2 && itemProto->Flags != 2048 && (itemProto->Class == ITEM_CLASS_WEAPON || itemProto->Class == ITEM_CLASS_ARMOR))
+	{
+		if (!GetOwner())
+		    return true;
+		GetOwner()->CreateWowarmoryFeed(2, itemid, guidlow, itemProto->Quality);
+	}
+
```
Comment (FrancescoBorzi) [inline src/server/game/Entities/Item/Item.cpp]: please fix your indentation

File: src/server/game/Entities/Player/Player.cpp [hunk 11]
```diff
@@ -5004,7 +5004,19 @@ void Player::DeleteFromDB(uint64 playerguid, uint32 accountId, bool updateRealmC
...
+			stmt = CharacterDatabase.GetPreparedStatement(CHAR_DEL_ARMORY_STATS);
+			stmt->setUInt32(0, guid);
+			trans->Append(stmt);
+			
+			stmt = CharacterDatabase.GetPreparedStatement(CHAR_DEL_FEED_LOG);
+			stmt->setUInt32(0, guid);
+			trans->Append(stmt);
+			trans->PAppend("DELETE FROM armory_character_stats WHERE guid = '%u'", guid);
+			trans->PAppend("DELETE FROM character_feed_log WHERE guid = '%u'", guid);
+			/** World of Warcraft Armory **/
```
Comment (FrancescoBorzi) [inline src/server/game/Entities/Player/Player.cpp]: please fix your indentation

**Comment by FrancescoBorzi** [comment]
@grimgravy thanks a lot for your PR, a few comments:

1) please read https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files#how-to-create-an-sql-update-file

2) some indentations need to be fixed

3) it looks like in your branch there are other things that aren't supposed to be in this PR, can you open a clean PR ? join our chat, we can help you

**Comment by grimgravy** [comment]
was testing the PR but was not to have many changes sorry

**Pull request closed without merging**